packages/app-store/salesforce/lib/CrmService.ts

```
@@ -857,10 +862,15 @@ export default class SalesforceCRMService implements CRM {
     });
 
     // Update the person record
-    await conn.sobject(personRecordType).update({
-      Id: contactId,
-      ...writeOnRecordBody,
-    });
+    await conn
+      .sobject(personRecordType)
+      .update({
+        Id: contactId,
+        ...writeOnRecordBody,
+      })
+      .catch((e) => {
+        this.log.error(`Error updating person record for contactId ${contactId}`, e);
```

This could be failing silently